src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
+            Ball randomBall = lottery.getRandomBall();
+
+            System.out.println(randomBall);
+        }
```

Comment:
please, delete regular empty lines

Reply:
sorry for that, I usually just separate statements and function calls by logical blocks with empty lines

Reply:
this method is quite simple for readability, so I recommend not to add empty lines